### PR TITLE
[164] Sign-Up Form Validation - Fullstack

### PR DESCRIPTION
**Pull request by kojusola**
## Task: `added signup form validation file`
- `validated the signup form using javascript`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done @kojusola 

But you defile the reason for using functions. `functions` are used to ensure code reusability. You wrote separate functions for things that look to similar instead of writing just one function that can handle all of them.

I will like for you to do a rework

**Comment by kojusola** [comment]
made changes to the function 
thank you

File: submissions/adeolaFafemi/dom/signUp.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+function regValidation(event, field,regex,error1,error2){
+    var fieldValue = field.value;
+    
+    if(fieldValue.length == 0)
+    {
+        field.nextElementSibling.innerText =error1;
+        event.preventDefault();
+        return false;
+    }
+    else
```
Comment (adaugochi) [inline submissions/adeolaFafemi/dom/signUp.js:37]: Too many nested `if else` statement. There is no need to have an `if else` inside the `else` block, Simply do an `else if` for the regex check and `else` if it returns true. Same goes for other places too

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by kojusola** [comment]
corrected it 
thanks

**Review by adaugochi — APPROVED** [review]

**Pull request merged**